doc/2_loading.md

```diff
@@ -184,7 +184,49 @@ val street = address.getString("street")
 val number = address.getInt("number")
 ```
 
-[//]: # (TODO loading vectors)
+### Reading vectors
```

Comment:
Should we add similar docs to https://github.com/datastax/spark-cassandra-connector/blob/master/doc/14_data_frames.md ?